hs/src/Reach/Report.hs

```diff
@@ -20,6 +20,8 @@ startReport :: Maybe String -> String -> IO (Report -> IO ())
 startReport mwho i = do
   startTime <- getCurrentTime
   cm <- lookupEnv "REACH_CONNECTOR_MODE" >>= maybe (pure "") pure
+  --- check env variable to track extension usage
+  vse <- lookupEnv "REACH_IDE" >>= maybe (pure "") pure
```

Comment:
@mattaudesse Can you add this to the CLI in this branch?

Reply:
Yup - will do.